### ISSUE TITLE
Comments in markdown are being rendered in the output.

### ISSUE THREAD

**Issue by redders6600** [tuchk4/storybook-readme#135]
Where I work we use a scaffold in our storybook readmes which includes a lot of pointers in html-style comments, such as:

```
# ActivityItem

<!-- Brief summary of what the component is, and what it's for. -->
```

Since upgrading to storybook 5, with storybook readme 5.0.1, the comments are rendered in our readmes:
![image](https://user-images.githubusercontent.com/2893500/54673199-b07bfc00-4af1-11e9-9817-70e83cebecf4.png)

**Comment by tuchk4** [comment]
@redders6600 thanks for reporting. 
Hope will fix it in `5.0.2` at the and is this week.

**Comment by redders6600** [comment]
@tuchk4 do you have an updated ETA for this? Appreciate it's probably not your main job, just trying to decide if we should find a workaround in the interim as this is preventing us migrating.

Sorry for not fixing & PRing myself - not too familiar with storybook so I think it would take me a year!

**Comment by tuchk4** [comment]
@redders6600 already done. Will be shipped in next release. Hope in few days.

**Comment by tuchk4** [comment]
published 5.0.2
